DownloadManager/Plugin.DownloadManager.Abstractions/IDownloadFile.cs

```diff
@@ -79,5 +79,11 @@ public interface IDownloadFile : INotifyPropertyChanged
         /// </summary>
         /// <value>The total bytes written.</value>
         float TotalBytesWritten { get; }
+
+        /// <summary>
+        /// Gets or sets wether this file can use Mobile network to download.
+        /// </summary>
+        /// <value>Indicates wether this file can use Mobile network for downloading</value>
+        bool MobileNetworkAllowed { get; set; }
```

Comment:
I'd take this private for the first. Do we have a solid documentation about what happens if you change this while downloading on the different platforms?